js/pronoun_text.js

```diff
@@ -0,0 +1,29 @@
+
+function initAddPronounText() {
+  var windowUrl = window.location.pathname;
+  var onProfileSettingsPage = (windowUrl.search(/\/profile\/settings/) != -1);
+
+  if (onProfileSettingsPage) {
+    textSpan = $( "#pronouns" ).parent().children().last();
```

Comment:
nit: should this be declared with `var` as well?

Reply:
Ah, yes, probably.